activestorage/lib/active_storage/attached/macros.rb

```diff
@@ -118,5 +126,23 @@ def purge_later
         ActiveRecord::Reflection.create(:has_many_attached, name, nil, { dependent: dependent }, self)
       )
     end
+
+    # specifies which file fields will be added for serialization
+    #
+    # example
+    # class Foo < ApplicationRecord
+    #   has_one_attached :image
+    #   has_many_attached :musics
+    #   serialize_attachments :image,:musics
```

Comment:
Minor doc comment, but do you mind adding space after `,`?

Reply:
okay